base/src/main/java/org/aya/core/pat/PatUnify.java

```diff
@@ -20,13 +20,13 @@
 public record PatUnify(@NotNull TermSubst lhsSubst, @NotNull TermSubst rhsSubst) {
   private void unify(@NotNull Pat lhs, @NotNull Pat rhs) {
     switch (lhs) {
+      default -> throw new IllegalStateException();
```

Comment:
Will this `default` dominate other `case`s?

Reply:
If it does the confluence checker will explode instantly after startup so I guess the answer is 'no' :D

Reply:
bors d=imkiva